packages/vx-tooltip/src/enhancers/withTooltip.tsx

```diff
@@ -1,28 +1,40 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 
-export const withTooltipPropTypes = {
-  tooltipOpen: PropTypes.bool,
-  tooltipLeft: PropTypes.number,
-  tooltipTop: PropTypes.number,
-  tooltipData: PropTypes.object,
-  updateTooltip: PropTypes.func,
-  showTooltip: PropTypes.func,
-  hideTooltip: PropTypes.func,
+type Props = {
```

Comment:
the types of the state would be the same as part of the `WithTooltipProvidedProps`, so I defined the state types by using it. If you think it would be better to have it separately - I'll fix it :)